bloom-housing/bloom
# fix: use lottery date in confirmation screen

## Issue Overview

This PR addresses #2814 


## Description

This PR updates the confirmation form step for applications to use the lottery date, not the application due date. Also relocates `getLotteryEvent` to shared helpers package.

## How Can This Be Tested/Reviewed?

Submit an application with an upcoming lottery, then verify the lottery date is displayed on the final confirmation screen.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
